src/frontend/src/tests/eth/derived/tokens.derived.spec.ts

```diff
@@ -0,0 +1,50 @@
+import * as ethEnv from '$env/networks.eth.env';
+import { ETHEREUM_TOKEN, SEPOLIA_TOKEN } from '$env/tokens.env';
+import { enabledEthereumTokens } from '$eth/derived/tokens.derived';
+import * as appContants from '$lib/constants/app.constants';
+import { testnetsStore } from '$lib/stores/settings.store';
+import { get } from 'svelte/store';
+import { expect } from 'vitest';
+
+describe('tokens.derived', () => {
+	describe('enabledEthereumTokens', () => {
+		beforeEach(() => {
+			vi.resetAllMocks();
+
+			testnetsStore.reset({ key: 'testnets' });
+
+			vi.spyOn(ethEnv, 'ETH_MAINNET_ENABLED', 'get').mockImplementation(() => true);
+
+			vi.spyOn(appContants, 'LOCAL', 'get').mockImplementation(() => false);
```

Comment:
In two other tests we mock local as following:

```
vi.mock(import('$lib/constants/app.constants'), async (importOriginal) => {
	const actual = await importOriginal();
	return {
		...actual,
		LOCAL: false
	};
});
```

Not for this PR but, maybe we should find a way to have a utils or something?

Reply:
Ah it's not applicable here, nevermind my comment. All good!